pom.xml

```diff
@@ -12,7 +12,7 @@
 	<url>https://github.com/deviceinsight/helm-maven-plugin</url>
 
 	<properties>
-		<kotlin.version>1.3.11</kotlin.version>
+		<kotlin.version>1.3.5</kotlin.version>
```

Comment:
I don't see how this is a version upgrade. :wink:

Reply:
My bad missed the 0 :S